crates/router/src/core/webhooks/incoming.rs

```diff
@@ -4,8 +4,9 @@ use actix_web::FromRequest;
 #[cfg(feature = "payouts")]
 use api_models::payouts as payout_models;
 use api_models::webhooks::{self, WebhookResponseTracker};
+use common_enums::EventClass;
 use common_utils::{errors::ReportSwitchExt, events::ApiEventsType, ext_traits::AsyncExt};
-use diesel_models::ConnectorMandateReferenceId;
+use diesel_models::{enums::EventObjectType, ConnectorMandateReferenceId};
```

Comment:
Let's avoid these direct imports please, we can qualify them with `enums::` instead.